ext/xmlrpc/xmlrpc.stub.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+function xmlrpc_encode($value): string {}
+
+function xmlrpc_decode(string $xml, string $encoding = "iso-8859-1") {}
+
+function xmlrpc_decode_request(string $xml, &$method, string $encoding = "iso-8859-1") {}
+
+function xmlrpc_encode_request(string $method, $params, array $output_options = UNKNOWN): string {}
```

Comment:
`$method` is nullable, and it seems that `null` can be returned as well, so:
```suggestion
function xmlrpc_encode_request(?string $method, $params, array $output_options = UNKNOWN): ?string {}
```